scripts/delete-limit-range.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+oc delete -f ./test-target/limit-range.yaml --namespace "$TNF_EXAMPLE_CNF_NAMES" --ignore-not-found
```

Comment:
Please add new line at the end of the script:
 https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

Reply:
Use descriptive flags in scripts: --filename instead of -f